services/quiz.js

```diff
@@ -1,6 +1,13 @@
 const Quiz = require('~/models/quiz')
 
 const quizService = {
+  getQuiz: async (match, skip, limit) => {
+    const items = await Quiz.find(match).skip(skip).limit(limit).sort({ updatedAt: -1 }).lean().exec()

```

Comment:
I think, you should use sort parameter for sorting

Reply:
Fixed